pkg/component/etcd/etcd/bootstrap.go

```diff
@@ -95,10 +109,28 @@ type bootstrapper struct {
 	etcdConfig           *config.ETCDConfig
 	image                string
 	imageVectorOverwrite *string
+	secretsManager       secretsmanager.Interface
+	secretNameServerCA   string
 	priorityClassName    string
 }
 
 func (b *bootstrapper) Deploy(ctx context.Context) error {
+	caSecret, found := b.secretsManager.Get(b.secretNameServerCA)
+	if !found {
+		return fmt.Errorf("secret %q not found", b.secretNameServerCA)
+	}
+
+	serverSecret, err := b.secretsManager.Generate(ctx, &secretsutils.CertificateSecretConfig{
+		Name:                        "etcd-druid-webhook-server-tls",
+		CommonName:                  fmt.Sprintf("%s.%s.svc", druidServiceName, b.namespace),
```

Comment:
Can't we use something simpler here, just `etcd-druid` maybe?

Reply:
made the above name change.